spec/OBD/OBD.vspec

```diff
@@ -68,85 +75,99 @@
   description: PID 04 - Engine load in percent - 0 = no load, 100 = full load
 
 - CoolantTemperature:
-  datatype: float
+  datatype: int16
```

Comment:
Example: No absolute need to use float in VSS, as only integer values in the range from -40 to +215 can be used